src/certificate.ts

```diff
@@ -23,6 +24,10 @@ export async function responseCertificate(
 
   renderSecondPage(doc, course.topics);
 
+  doc.addPage();
+
+  await renderThirdPage(doc, talent, talent.capstoneProject);
```

Comment:
Is it required to have 3 instead of 2 arguments? `capstoneProject` is already part of `talent`.